docs/demo.rst

```diff
@@ -217,6 +217,156 @@ using:
    variant, and drill down into the reads in question programatically.
    values as parameters which have sensible defaults.
 
+Host the Thousand Genomes VCF
+=============================
+
+The GA4GH reference server uses a registry of files and URL’s to
+populate its data repository. This registry is a SQLite database that is
+read when the server is started. In this tutorial we will use the
+command-line client to create a registry similar to that used by
+1kgenomes.ga4gh.org. Your system should have samtools installed, and at
+least 30GB to host the VCF and reference sets.
+
+Repo administrator CLI
+----------------------
+
+The CLI has methods for adding and removing Feature Sets, Read Group
+Sets, Variant Sets, etc. Before we can begin adding files we must first
+initialize an empty database. The directory should be readable and
+writable by the current user, as well as the user running the server.
+
+.. code-block:: bash
+
+    ga4gh_repo init registry.db
+
+This command will create a file ``registry.db`` in the current working
+directory. This file should stay relatively small (a few MB for
+thousands of files).
+
+Now we will add a dataset to the registry, which is a logical container
+for the genomics data we will later add. You can optionally provide a
+description using the ``--description`` flag.
+
+.. code-block:: bash
+
+    ga4gh_repo add-dataset registry.db 1kgenomes --description "Variants from the 1000 Genomes project and GENCODE genes annotations"
+
+Add a Reference Set
+-------------------
+
+It is possible for a server to host multiple reference assemblies. Here
+we will go through all the steps of downloading and adding the FASTA
+used for the Thousand Genomes VCF.
+
+.. code-block:: bash
+
+    wget ftp://ftp.1000genomes.ebi.ac.uk//vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz
+
+This file is provided in ``.gz`` format, which we will decompress, and
+then with samtools installed on the system, recompress it using
+``bgzip``.
+
+.. code-block:: bash
+
+    gunzip hs37d5.fa.gz
+    bgzip hs37d5.fa
+
+This may take a few minutes depending on your system as this file is
+around 3GB. Next, we will add the reference set.
+
+.. code-block:: bash
+
+    ga4gh_repo add-referenceset registry.db /full/path/to/hs37d5.fa.gz -d “NCBI37 assembly of the human genome” --ncbiTaxonId 9606 --name NCBI37 --sourceUri "ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz"
```

Comment:
Perhaps add some line breaks in this command to make it a bit more readable? I.e.
```bash
$ ga4gh_repo add-referenceset registry.db /full/path/to/hs37d5.fa.gz \
       -d “NCBI37 assembly of the human genome” --name NCBI37 \
       # etc
``